<launch>
    <!-- <arg name="config" default="$(find langnav)/config/config.yaml" />
    <rosparam file="$(arg config)" command="load" /> -->
    
    <node ns="planning" pkg='path_planner' type='path_planner' name='path_planner'>
        <!-- <param name="/margin" type="int" value="3" /> -->
        <param name="/view_range" type="int" value="10" />
        <param name="/cost_co" type="double" value="0.2" />
        <param name="/enable_clicked_set" type="bool" value="true" />
    </node>
    
    <!-- visulaization -->
    <arg name="rviz" default="false" />
    <group if="$(arg rviz)">
        <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find path_planner)/rviz/debug.rviz" />
    </group>
</launch>
